test/module/irohad/ametsuchi/postgres_executor_test.cpp

```diff
@@ -2611,7 +2611,53 @@ namespace iroha {
       auto kv = sql_query->getAccountDetail(account_id);
       ASSERT_TRUE(kv);
       ASSERT_EQ(kv.get(), R"({"id@domain": {"key": "value"}})");
-    }
 
-  }  // namespace ametsuchi
-}  // namespace iroha
+      class SetSettingValueTest : public CommandExecutorTest {
+        void SetUp() override {
+          CommandExecutorTest::SetUp();
+        }
+      };
+
+      /**
+       * @given command
+       * @when trying to insert the setting value by the key
+       * @then record with the key has the value
+       */
+      TEST_F(SetSettingValueTest, InsertSettingValue) {
+        std::string key = "maxDesc";
+        std::string value = "255";
+        CHECK_SUCCESSFUL_RESULT(execute(
+            *mock_command_factory->constructSetSettingValue(key, value), true));
+
+        auto setting_value = sql_query->getSettingValue(key);
+        ASSERT_TRUE(setting_value);
+        ASSERT_STRCASEEQ(setting_value.get().c_str(), value.c_str());
```

Comment:
Why not `ASSERT_EQ` for strings? same below.

Reply:
Fixed